personalities/sctfe/config.go

```diff
@@ -185,22 +178,7 @@ func ValidateLogConfig(cfg *configpb.LogConfig, origin string, projectID string,
 	}
 
 	// Validate the time interval.
-	start, limit := cfg.NotAfterStart, cfg.NotAfterLimit
-	if start != nil {
-		vCfg.NotAfterStart = &time.Time{}
-		if err := start.CheckValid(); err != nil {
-			return nil, fmt.Errorf("invalid start timestamp: %v", err)
-		}
-		*vCfg.NotAfterStart = start.AsTime()
-	}
-	if limit != nil {
-		vCfg.NotAfterLimit = &time.Time{}
-		if err := limit.CheckValid(); err != nil {
-			return nil, fmt.Errorf("invalid limit timestamp: %v", err)
-		}
-		*vCfg.NotAfterLimit = limit.AsTime()
-	}
-	if start != nil && limit != nil && (*vCfg.NotAfterLimit).Before(*vCfg.NotAfterStart) {
+	if notAfterStart != nil && notAfterLimit != nil && (notAfterLimit).Before(*notAfterStart) {
```

Comment:
This looks like a change in behaviour; previously it looks like you could configure the log to enforce either `start`, `limit`, or both, but now it will silently ignore one of these values unless both are non-`nil`). Was that intended?

Reply:
The values are passed on directly in the ValidatedConfig directly above. The old code would stage them in the LogConfig, then set them in ValidatedConfig manually, after it did a prototimestamp to time.Time conversion. The nil checks here are only to make sure that we can call .`Before()` safely.

Reply:
Ah gotcha, thanks!